Common/Exceptions/StandartPresetException.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Runtime.Serialization;
+using System.Text;
+
+namespace Common.Exceptions
+{
+    public class StandartPresetException : Exception
```

Comment:
bad name. when this exception might be thrown?

Reply:
when user tries to delete a standart preset